Cargo.toml

```
@@ -12,5 +12,6 @@ members = [
     "nft-marketplace/nft",
     "non-fungible-token",
     "ping",
-    "escrow"
+    "escrow",
+    "supply-chain"
```

The common style is to add a comma always.
```suggestion
    "supply-chain",
```